tests/ui/unnecessary_join.rs

```diff
@@ -0,0 +1,35 @@
+#![warn(clippy::unnecessary_join)]
```

Comment:
```suggestion
// run-rustfix

#![warn(clippy::unnecessary_join)]
```

To get rustfix to check that the suggestion works.